src/web/utils/AwaitTypesafe.tsx

```diff
@@ -11,3 +11,24 @@ export type AwaitProps<T> = {
 };
 
 export const AwaitTypesafe = RRAwait as <T>(props: AwaitProps<T>) => React.JSX.Element;
+
+/*
```

Comment:
Maybe it's worth using JSDoc notation here?
```
/**
 * Accepts an object consisting of keys and values that are promises, passes them through
 * Promise.all, and destructures the result back to the Promise results mapped to the original keys.
 */
```

Reply:
Done.